### DISCUSSION TITLE
It's possible use the latest version of SDK 2.0 in Ubuntu 22.04 with the camera SR305?

### DISCUSSION THREAD

**Discussion by HengyiDu** [realsenseai/realsense-ros#3126] [Q&A]
I have the camera SR305 and Ubuntu 22.04 and I tried to connect with ROS2 Humble using the latest version SDK2.0, but starting the camera node with comand ros2 run can not find the camera node. I check that the camera is connected to the laptop and all the package is dowloaded correctly. The laptop also detect the camera in the list of usb connections. Any solution for this camera (SR305) or it's just the camera can be used with 22.04 using the corresponding version of SDK.

**Comment by MartyG-RealSense** [comment]
Hi @HengyiDu  The issue is not related to Ubuntu 22.04.  The SR300 and SR305 are retired camera models and support for them has been removed from the librealsense SDK and the ROS2 wrapper - please see https://github.com/IntelRealSense/realsense-ros/pull/2841

Support for Humble was first added in ROS2 wrapper **4.51.1**, and librealsense **2.51.1** still supported SR300 / SR305.  So using this combination of librealsense and wrapper versions may be the best solution for you.
